dot/parachain/availability-distribution/session_cache_test.go

```diff
@@ -109,6 +110,121 @@ func TestReportBadValidators(t *testing.T) {
 	}
 }
 
+func TestGetAuthorityID(t *testing.T) { //nolint:tparallel
+	t.Parallel()
+
+	var (
+		ctrl        *gomock.Controller
+		runtimeMock *MockInstance
+		cache       *LRUSessionCache
+
+		relayParent = common.Hash{0x01}
+	)
+
+	setup := func(t *testing.T) {
+		ctrl = gomock.NewController(t)
+		defer ctrl.Finish()
+
+		runtimeMock = NewMockInstance(ctrl)
+		cache = NewLRUSessionCache(nil)
+	}
+
+	testCases := []struct {
+		description    string
+		validatorIndex parachaintypes.ValidatorIndex
+		setUpRuntime   func(t *testing.T, rt *MockInstance)
+		expectedAuthID parachaintypes.AuthorityDiscoveryID
+		errExpected    bool
+	}{
+		{
+			description:    "session_index_for_child_fails",
+			validatorIndex: 0,
+			setUpRuntime: func(t *testing.T, rt *MockInstance) {
+				rt.EXPECT().
+					ParachainHostSessionIndexForChild().
+					Return(parachaintypes.SessionIndex(0), errors.New("fail")).
+					Times(1)
+			},
+			expectedAuthID: parachaintypes.AuthorityDiscoveryID{},
+			errExpected:    true,
+		},
+		{
+			description:    "session_info_fails",
+			validatorIndex: 0,
+			setUpRuntime: func(t *testing.T, rt *MockInstance) {
+				rt.EXPECT().
+					ParachainHostSessionIndexForChild().
+					Return(parachaintypes.SessionIndex(0), nil).
+					Times(1)
```

Comment:
No need to write the `Times(1)` if its expected to call only once.
Can you remove  `.Times(1)` from everywhere you have used?

Reply:
done